test/D2L.Security.OAuth2.WebApi.UnitTests/packages.config

```diff
@@ -4,10 +4,56 @@
   <package id="FluentAssertions" version="3.2.2" targetFramework="net451" />
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net451" />
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net451" />
+  <package id="Microsoft.NETCore.Platforms" version="1.0.1" targetFramework="net47" />
+  <package id="Microsoft.Win32.Primitives" version="4.0.1" targetFramework="net47" />
   <package id="Moq" version="4.2.1409.1722" targetFramework="net451" />
+  <package id="NETStandard.Library" version="1.6.0" targetFramework="net47" />

```

Comment:
Hmm... this one makes me suspicious. Would love to actually be NETStandard, but I don't believe that's the case as of yet... what brings this in?